src/devices/README.md

```diff
@@ -6,15 +6,23 @@ Our vision: the majority of .NET bindings are written completely in .NET languag
 
 ## Binding Index
 
-* [BMP280 -- digital pressure sensor](Bmp280/README.md)
-* [Dhtxx -- digital-output relative humidity & temperature sensor module](Dhtxx/README.md)
-* [Hcsr04 -- ultrasonic ranging module](Hcsr04/README.md)
-* [CharacterLcd](CharacterLcd/README.md)
-* [Max7219 -- LED matrix driver](Max7219/README.md)
-* [Mcp23xxx -- I/O expander](Mcp23xxx/README.md)
-* [Mcp3008 -- analog to digital converter](Mcp3008/README.md)
-* [Pca95x4 -- I2C GPIO expander](Pca95x4/README.md)
-* [Si7021 -- temperature and humidity sensor](Si7021/README.md)
+* [ADS1115 -- Analog-to-Digital Converter](Ads1115/README.md)
```

Comment:
thanks for fixing this. Hope we will eventually add some script for generating this